### ISSUE TITLE
TV Guide Fails To Load (New Install)

### ISSUE THREAD

**Issue by TypeR03260** [echocoderkodi/repository.echo#5]
Hi,

I've installed your TV guide maybe 50 times with minimal issues, over the past 24 hours though I haven't been able to get the guide to update on it's first run after installation. Also cannot download a skin in the settings menu.

Have you had any other reported issues?
Any ideas what I can do to get around this issue?

Thanks your your work on the Wizard and TV Guide, it's great work and the Wizard is my number one KODI app.

Best Regards

Alex
[kodi.zip](https://github.com/echocoderkodi/repository.echo/files/830608/kodi.zip)

**Comment by wullbruce** [comment]
I the same problem. Hope there is a fix 

Wonderful work guys

**Comment by FuglyNut** [comment]
Same issue, but it's not giving me an error message or error log.  I'm in Kodi 17 btw.  When I click information, under the info, it says version 1.11 is installed, status says disabled, even though I didn't disable it.  I tried to disable and really after I saw that, thinking that might be the issue, but no lucky.

When I try to run the program a download and install pop up comes and goes quickly and then nothing... no error, nada.  I'm pretty sure it's related to the disabled status, but I'm definitely no expert. :)

Any helped would be appreciated, but not expected. ;)
